app/scripts/modules/core/pipeline/config/triggers/webhook/webhookTrigger.module.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.trigger.webhook', [
+    require('../../../../../core/config/settings.js'),
+  ])
+  .config(function (pipelineConfigProvider) {
+    pipelineConfigProvider.registerTrigger({
+      label: 'Webhook',
+      description: 'Executes the pipeline on a Webhook POST',
+      key: 'webhook',
+      controller: 'WebhookTriggerCtrl as ctrl',
+      controllerAs: 'vm',
+      templateUrl: require('./webhookTrigger.html'),
+      popoverLabelUrl: require('./webhookPopoverLabel.html'),
+    });
+  })
+  .controller('WebhookTriggerCtrl', function (trigger, $scope) {
```

Comment:
I haven't tried this, but it looks like Gate has a webhook endpoint that mirrors (and passes through to) Echo's:
https://github.com/spinnaker/gate/blob/master/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EventController.groovy

You should be able to call it the same way you'd call the webhook endpoint in Echo, just replacing the base URL, e.g.

```
http://echo.mySpinnaker.com/webhooks/type/source
```
becomes
```
http://gate.mySpinnaker.com/webhooks/type/source
```